src/components/AttachmentImageViewer.vue

```diff
@@ -28,25 +26,16 @@ export default {
 
 <style lang="scss" scoped>
 ::v-deep .modal-wrapper {
-	&--large {
-		.modal-container,
-		.modal__attachment {
-			max-width: 85vw !important;
-			max-height: 90vh !important;
-		}
-	}
-
-	&--full {
-		.modal-container,
-		.modal__attachment {
-			max-width: 95vw !important;
-			max-height: 95vh !important;
-		}
+	.modal-container {
+		background-color: unset !important;
+		box-shadow: none !important;
```

Comment:
We could also decide to keep the background...

Reply:
Maybe makes sense espwcially for mobile...

Reply:
yes, we should keep the background otherwise logos with just text, it will e hard to tell.